src/styles/blocks/card.scss

```
@@ -0,0 +1,135 @@
+$white-color-primary: #fff;
+$black-color-primary: #060b35;
+$neutral-color-primary: #616070;
+$accent-color-primary: #00acdc;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+body,
+p,
+h2 {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  background-color: #{$white-color-primary};
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #{$black-color-primary};
+    display: flex;
+    word-break: break-word;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #{$neutral-color-primary};
+    display: flex;
+    margin-top: 4px;
+  }
+
+  &__images {
+    display: flex;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__global {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    height: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    color: #{$black-color-primary};
+    align-items: center;
+    display: flex;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #{$neutral-color-primary};
+  }
+
+  &__value {
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #{$white-color-primary};
+    border-radius: 5px;
+    background-color: #{$accent-color-primary};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    padding: 11px 0;
+
+    &:hover {
+      border: 1px solid #{$accent-color-primary};
+      background-color: #{$white-color-primary};
+      color: #{$accent-color-primary};
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background: center no-repeat url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    padding: 0;
+    display: flex;
+  }
+}
+
+.stars--2 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--2 :nth-child(-n+3),
+.stars--2 :nth-child(-n+4) {
```

suppose it should looks like
```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
```